src/guides/v2.3/reference/mftf/action-groups.md

```diff
@@ -0,0 +1,27 @@
+---
+group: mftf
+---
+
+# MFTF action group reference
+
+Action groups are important building blocks for quickly creating tests for the Magento Functional Testing Framework.
+This page lists all current action groups so developers can see what is available to them.
+
+## Action group list
+
```

Comment:
It would be nice to have here some explanation of what this list contains such as:
_This list is grouped by a module. One module can have multiple XML files each of which contains one or more action groups. To learn more about action groups, refer to [Action groups](https://devdocs.magento.com/mftf/docs/test/action-groups.html)_ 

The [Action groups](https://devdocs.magento.com/mftf/docs/test/action-groups.html) topics should contain a link to this reference.

Reply:
Similar for the Functional tests topic.